localization/languages/cs.json

```diff
@@ -101,7 +101,7 @@
         "settingsAppearanceHeading": "Vzhled",
         "settingsDarkModeToggle": "Použít tmavý režim",
         "settingsHistoryButtonToggle": "Zobrazit tlačítko zpět",
-        "settingsSwipeNavigationHeading": "Swipe navigace",
+        "settingsadditionalFeaturesHeading": null, //missing translation
```

Comment:
Shouldn't this be `settingsAdditionalFeaturesHeading` instead?

Reply:
Yup, fixed in 8c0d405dac066e941f87f5865de6a20360e214aa.